src/models/models.ts

```diff
@@ -1,4 +1,6 @@
-export interface Error {
```

Comment:
better to move and rename this file to interfaces

Reply:
remaned and moved this to interfaces